clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -64,6 +64,10 @@
             {
                 "command": "cobol-lsp.commentLine.uncomment",
                 "title": "Remove COBOL Line Comment"
+            },
+            {
+                "command": "cobol-lsp.clear.copybooks.cache",
```

Comment:
@zeibura Is this command ok ?

This command clears all the downloaded copybook from .c4z/copybooks

Reply:
Yes, F1 menu

Reply:
Updated to
```
{
                "command": "cobol-lsp.clear.downloaded.copybooks",
                "title": "clear downloaded copybooks"
            }
```